composer.json

```diff
@@ -12,42 +12,40 @@
     ],
     "require": {
         "php": ">=7.1",
-        "fsi/admin-bundle" : "^3.0@dev",
+        "fsi/admin-bundle" : "^3.0",
         "symfony/console": "^2.6|^3.0|^4.0",
         "symfony/dependency-injection": "^2.7|^3.0|^4.0",
         "symfony/form": "^2.7|^3.0|^4.0",
         "symfony/framework-bundle": "^2.7|^3.0|^4.0",
+        "symfony/property-access": "^2.6|^3.0|^4.0",
         "symfony/security": "^2.6|^3.0|^4.0",
         "symfony/security-bundle": "^2.6|^3.0|^4.0",
         "symfony/swiftmailer-bundle": "^2.4|^3.0|^4.0",
-        "symfony/property-access": "^2.6|^3.0|^4.0",
         "symfony/twig-bundle": "^2.7|^3.0|^4.0",
         "symfony/validator": "^2.6|^3.0|^4.0",
         "doctrine/orm": "^2.5",
         "doctrine/doctrine-bundle": "^1.5",
         "paragonie/random_compat": "^1.1"
     },
     "require-dev": {
-        "symfony/dom-crawler": "^2.5|^3.0|^4.0",
-        "symfony/class-loader": "^2.4|^3.0|^4.0",
-        "symfony/monolog-bridge": "^2.4|^3.0|^4.0",
-        "symfony/monolog-bundle": "^2.6|^3.0|^4.0",
-        "symfony/routing": "^2.7|^3.0|^4.0",
-        "symfony/var-dumper": "^2.7|^3.0|^4.0",
-        "phpspec/phpspec": "^3.0|^4.0",
-        "phpspec/prophecy": "^1.7.2",
         "behat/behat": "^3.3",
-        "behat/symfony2-extension": "^2.1",
         "behat/mink": "^1.7",
-        "behat/mink-extension": "^2.2",
         "behat/mink-browserkit-driver": "^1.3",
+        "behat/mink-extension": "^2.2",
         "behat/mink-selenium2-driver": "^1.3",
-        "sensiolabs/behat-page-object-extension": "^2.0",
+        "behat/symfony2-extension": "^2.1",
         "bossa/phpspec2-expect": "^2.3|^3.0",
-        "fsi/datagrid": "^2.0@dev",
-        "fsi/datagrid-bundle": "^2.0@dev",
-        "fsi/datasource": "^2.0@dev",
-        "fsi/datasource-bundle": "^2.0@dev"
+        "fsi/datagrid-bundle": "^2.0",
+        "fsi/datasource-bundle": "^2.0",
```

Comment:
`fsi/datagrid-bundle` and `fsi/datasource-bundle` does not need to be specified here because they are required by `fsi/admin-bundle` with the same version constraints